### PR TITLE
[1LP][RFR][BLOCKED] Tests for the network and storage devices links

### PR DESCRIPTION
**Pull request by skovic**
This PR contains tests that verify that the storage and network devices links on the physical server summary page work correctly.

Depends on https://github.com/ManageIQ/wrapanapi/pull/252 and https://github.com/ManageIQ/integration_tests/pull/7221

### CODE REVIEW

**Comment by skovic** [comment]
@mshriver Could you take a look at this PR when you get a chance? Thanks

File: cfme/common/physical_server_views.py [hunk 1]
```diff
@@ -221,3 +221,19 @@ class PhysicalServersView(ComputePhysicalInfrastructureServersView):
...
         return (self.in_compute_physical_infrastructure_servers and
                 self.title.text == "Physical Servers")
+
+
+class PhysicalServerNetworkDevicesView(ComputePhysicalInfrastructureServersView):
+    """Represents the Network Devices page"""
+
+    @property
+    def is_displayed(self):
+        return "Network Devices" in self.title.text and self.logged_in_as_current_user
```
Comment (rlbabyuk) [inline cfme/common/physical_server_views.py]: Required: better use self.in_compute_physical_infrastructure_servers as it also covers self.logged_in_as_current_user

File: cfme/common/physical_server_views.py [hunk 2]
```diff
@@ -221,3 +221,19 @@ class PhysicalServersView(ComputePhysicalInfrastructureServersView):
...
+
+
+class PhysicalServerStorageDevicesView(ComputePhysicalInfrastructureServersView):
+    """Represents the Storage Devices page"""
+
+    @property
+    def is_displayed(self):
+        return "Storage Devices" in self.title.text and self.logged_in_as_current_user
```
Comment (rlbabyuk) [inline cfme/common/physical_server_views.py]: self.in_compute_physical_infrastructure_servers same here

File: cfme/tests/physical_infrastructure/ui/test_physical_server_details.py [hunk 3]
```diff
@@ -52,6 +52,28 @@ def test_physical_server_details_dropdowns(physical_server):
     assert "Timelines" in monitoring_items
 
 
+def test_network_devices(physical_server):
+    """Navigate to the Network Devices page and verify that the page is displayed"""
+
+    if physical_server.num_network_devices() != "0":
```
Comment (rlbabyuk) [inline cfme/tests/physical_infrastructure/ui/test_physical_server_details.py]: Optional: more compact variant to remove steps duplication
devices_num = physical_server.num_network_devices()
network_device_view = navigate_to(physical_server, 'NetworkDevices')
assert (
    network_device_view.is_displayed if devices_num !='0' else not network_device_view.is_displayed
)

**Comment by dajoRH** [comment]
Lint report for commit 0563a0796b0e6bafae2d9f04bfab8ee046d3a14b:

`cfme/common/physical_server_views.py`:
- :bangbang: Line 231:101: **E501** *line too long (103 > 100 characters)*
- :bangbang: Line 239:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by skovic** [comment]
@rlbabyuk I believe I have addressed all of your comments. Please let me know if I missed anything. Thanks

**Comment by mshriver** [comment]
It looks like this PR actually includes the code from #7221, if that's the case please close #7221 and I'll merge this.

**Comment by mshriver** [comment]
Added [BLOCKED] to title since we're waiting on a wrapanapi PR + release for supporting this.

**Comment by skovic** [comment]
@mshriver I've closed  #7221 and it looks like the wrapanapi PR has been merged. Can we go ahead and merge this PR? Thanks

**Comment by mshriver** [comment]
We've got the wrapanapi release now that includes the needed methods.

**Pull request merged**